run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sleep 3600
```

Comment:
wont this kill the container after 1 hour? also, missing new line

Reply:
Yes, and will allow testing on a running container.  For now it's a placeholder so we can decide what to do about it (implement a real script that takes care of running things or leave it to the charm)